extensions/database-documentation/.vscodeignore

```diff
@@ -0,0 +1,10 @@
+.vscode
+.ads-test
+out/test
+src
+.gitignore
+vsc-extension-quickstart.md
```

Comment:
for this file and the .gitignore file, please only leave the entries that are relevant to your extension.